composer.json

```diff
@@ -20,7 +20,8 @@
         "symfony/security-bundle": "5.0.*",
         "symfony/string": "5.0.*",
         "symfony/validator": "5.0.*",
-        "symfony/yaml": "5.0.*"
+        "symfony/yaml": "5.0.*",
+        "symfonycasts/reset-password-bundle": "^1.1"
```

Comment:
not in this PR. And I think you are not allowed to use external library on this project